README.md

```diff
@@ -139,7 +140,23 @@ Options:
 	--loglevel            Set the logging level (error|warn|info|verbose|silly).  [string] [default: "info"]
 ```
 
+#### versions
+
+`ui5 versions [options]` show the versions of all UI5 Build and Development Tooling packages. It will check the local-installed version first. If not found, then check the global-installed version.
```

Comment:
`It will check the local-installed version first. If not found, then check the global-installed version.` can be removed I guess?